### PR TITLE
Add labels and helper text to form components

### PR DESCRIPTION
**Pull request by youda97**
Sort of touches on #497

### CODE REVIEW

File: src/combobox/combobox.stories.ts [hunk 1]
```diff
@@ -15,11 +15,13 @@ storiesOf("Combobox", module)
 	.addDecorator(withKnobs)
 	.add("Basic", () => ({
 		template: `
-			<ibm-combo-box [items]="items" (selected)="selected($event)">
+			<ibm-combo-box [label]="label" [helperText]="helperText" [items]="items" (selected)="selected($event)">
```
Comment (cal-smith) [inline src/combobox/combobox.stories.ts]: These attrs should probably get split onto newlines (same for the rest of the stories)

File: src/combobox/combobox.component.ts [hunk 2]
```diff
@@ -31,55 +30,60 @@ import { filter } from "rxjs/operators";
...
-			role="button"
-			class="bx--list-box__field"
-			tabindex="0"
-			type="button"
-			aria-label="close menu"
-			aria-haspopup="true"
-			(click)="toggleDropdown()">
+		<label [for]="id" class="bx--label">{{label}}</label>
+		<div class="bx--form__helper-text">{{helperText}}</div>
+		<div class="bx--combo-box bx--list-box" [ngClass]="{'bx--multi-select' : type === 'multi'}">
```
Comment (cal-smith) [inline src/combobox/combobox.component.ts]: This should be split onto newlines

File: src/dropdown/dropdown.component.ts [hunk 3]
```diff
@@ -29,7 +30,10 @@ import { DropdownService } from "./dropdown.service";
 @Component({
 	selector: "ibm-dropdown",
 	template: `
+	<label [for]="id" class="bx--label">{{label}}</label>
```
Comment (zvonimirfras) [inline src/dropdown/dropdown.component.ts]: Again, `innerHtml` for these two?

File: src/slider/slider.stories.ts [hunk 4]
```diff
@@ -1,6 +1,6 @@
 import { storiesOf, moduleMetadata } from "@storybook/angular";
 import { action } from "@storybook/addon-actions";
-import { withKnobs, number, text, boolean } from "@storybook/addon-knobs/angular";
+import { withKnobs, number, text, boolean, select } from "@storybook/addon-knobs/angular";
```
Comment (zvonimirfras) [inline src/slider/slider.stories.ts]:
```suggestion
import {
	withKnobs,
	number,
	text,
	boolean,
	select
} from "@storybook/addon-knobs/angular";
```

File: src/timepicker-select/timepicker-select.component.ts [hunk 5]
```diff
@@ -10,7 +10,7 @@ import { Select } from "../select/select.component";
 @Component({
 	selector: "ibm-timepicker-select",
 	template: `
-			<label *ngIf="!skeleton" [attr.for]="id" class="bx--label bx--visually-hidden">{{label}}</label>
+			<label *ngIf="!skeleton" [attr.for]="id" class="bx--label bx--visually-hidden" [innerHtml]="label"></label>
```
Comment (cal-smith) [inline src/timepicker-select/timepicker-select.component.ts]: Same issue as the other PR, if we want to allow arbitrary template content, we should take a template as an input - not raw HTML

File: src/content-switcher/content-switcher.component.ts [hunk 6]
```diff
@@ -47,7 +47,7 @@ export class ContentSwitcher implements AfterViewInit {
 	/**
 	 * aria-label for the content switcher. Should be set to something descriptive
 	 */
-	@Input() label = "content switcher";
+	@Input() ariaLabel = "content switcher";
```
Comment (zvonimirfras) [inline src/content-switcher/content-switcher.component.ts]: This is, technically, a breaking change.

We should deprecate `label` instead, wrap it with getters and setters to `ariaLabel` and add warning messages in them.

File: src/radio/radio.stories.ts [hunk 7]
```diff
@@ -12,21 +12,28 @@ storiesOf("Radio", module).addDecorator(
...
-				[checked]="true">
-				zero
-			</ibm-radio>
-			<ibm-radio *ngFor="let radio of manyRadios"
-				[value]="radio.num"
-				[disabled]="radio.disabled">
-				{{radio.num}}
-			</ibm-radio>
-		</ibm-radio-group>
+		<fieldset class="bx--fieldset">
```
Comment (cal-smith) [inline src/radio/radio.stories.ts]: thought: `fieldset` could be a component 🤔

File: src/number-input/number.component.ts [hunk 8]
```diff
@@ -1,4 +1,4 @@
-import { Component, Input, HostBinding, EventEmitter, Output } from "@angular/core";
+import { Component, Input, HostBinding, EventEmitter, Output, TemplateRef } from "@angular/core";
```
Comment (cal-smith) [inline src/number-input/number.component.ts]:
```suggestion
import { 
	Component, 
	Input, 
	HostBinding, 
	EventEmitter, 
	Output, 
	TemplateRef 
} from "@angular/core";
```

Comment (zvonimirfras) [inline src/number-input/number.component.ts]:
```suggestion
import {
	Component,
	Input,
	HostBinding,
	EventEmitter,
	Output,
	TemplateRef
} from "@angular/core";
```

File: src/radio/radio.stories.ts (hunk 7 above)
Comment (youda97) [reply src/radio/radio.stories.ts]: I mean it's only used once so I don't see a need to

Comment (cal-smith) [reply src/radio/radio.stories.ts]: _here_ it is, but it's a pretty useful form related element: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/fieldset

Comment (cal-smith) [reply src/radio/radio.stories.ts]: Obviously we don't need any more change in this PR, but something to consider

**Pull request merged**